packages/client/ui/react-ui/src/providers/CrossmintWalletProvider.tsx

```diff
@@ -1,11 +1,11 @@
 import { useCrossmint } from "@/hooks";
 import { ReactNode, createContext, useEffect, useMemo, useState } from "react";
 
-import { EVMSmartWallet, SmartWalletError, SmartWalletSDK } from "@crossmint/client-sdk-smart-wallet";
+import { EVMSmartWallet, SmartWalletChain, SmartWalletError, SmartWalletSDK } from "@crossmint/client-sdk-smart-wallet";
 
 export type CrossmintWalletConfig = {
     type: "evm-smart-wallet";
-    defaultChain: "polygon-amoy" | "base-sepolia" | "optimism-sepolia" | "arbitrum-sepolia";
+    defaultChain: SmartWalletChain;
```

Comment:
Do we have a good error if you pass a chain that's not compatible with the environment your API key is set to?

Reply:
No, mind if I add that as a follow up?

Reply:
will do so unless you object!